.ci/pipeline/ci.yml

```diff
@@ -143,6 +165,8 @@ jobs:
   variables:
     release.dir: '__release_lnx_gnu'
     platform.type : 'lnx32e'
+    OPENBLAS_VERSION: 'v0.3.27'
```

Comment:
do we need per pipeline definitions vs centralized global versioning for TBB and OPENBLAS?

Reply:
I had thought about that, and decided to go for per pipeline for now. The use case being errata in the build for compiler / OS / architecture. But that is not the case at the moment (including for the RISC-V build I have locally), so I could put these as global for now, and we can make them per-pipeline if the need arises. What is your preference here?

Reply:
i think ether would work. We can keep as is